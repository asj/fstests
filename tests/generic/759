#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
#
# FS QA Test No. 759
#
# fsx exercising reads/writes from userspace buffers
# backed by hugepages
#
. ./common/preamble
_begin_fstest rw auto quick

. ./common/filter

_require_test
_require_thp
_require_hugepage_fsx

_run_hugepage_fsx -N 10000            -l 500000
_run_hugepage_fsx -N 10000  -o 8192   -l 500000
_run_hugepage_fsx -N 10000  -o 128000 -l 500000

status=0
exit
