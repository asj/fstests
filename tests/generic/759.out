QA output created by 759
fsx -N 10000 -l 500000 -h
fsx -N 10000 -o 8192 -l 500000 -h
fsx -N 10000 -o 128000 -l 500000 -h
